examples/dtls-echo/Makefile

```diff
@@ -49,6 +49,8 @@ CFLAGS += -DDEVELHELP
 # NOTE: Add the package for TinyDTLS
 USEPKG += tinydtls
 
+USEMODULE += tinydtls
```

Comment:
hm, usepkg is enough, but always necessary to even consider the pkg dir. the usemodule should be redundant here. does this build without ?